data/air.yaml

```diff
@@ -0,0 +1,177 @@
+description: |-
```

Comment:
Following up on discussion in commit 89157a4 (which is not displayed outside of commit):

I believe it would be preferable to add the description to the initial source `air.inp` instead of adding it by hand. I.e. `ck2yaml` will generate this field automatically (I did as much in 5743959).